ci/values/k3s-plugin-server-split.yaml

```diff
@@ -0,0 +1,38 @@
+cloud: k3s
+
+image:
+  # :TODO: Remove this once 1.37.0 is out
+  tag: latest
```

Comment:
Can we use a sha here to reduce any CI flakiness might be created by PostHog images?